src/containers/EditListingPage/EditListingPage.js

```diff
@@ -146,6 +147,7 @@ export const EditListingPageComponent = props => {
           desktopClassName={css.desktopTopbar}
           mobileClassName={css.mobileTopbar}
         />
+        <UserNav selectedPageName="ContactDetailsPage" listing={listing} />
```

Comment:
I’d like to see how this navigation works. Originally we wanted to reduce possible links on EditListingWizard so that people would not navigate away from it but complete it.